helm/dagster/charts/dagster-user-deployments/templates/serviceaccount.yaml

```
@@ -10,4 +10,8 @@ kind: ServiceAccount
 metadata:
   name: {{ include "dagsterUserDeployments.serviceAccountName" . }}
   labels: {{ include "dagster.labels" . | nindent 4 }}
+  {{- with .Values.serviceAccount.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
```

This `with` statement is overkill imo 

```suggestion
  annotations: {{- .Values.serviceAccount.annotations | toYaml | nindent 4 }}
```

---

Yep, that's definitely better.